### ISSUE TITLE
Console error while using resizeHandler

### ISSUE THREAD

**Issue by JananiGunasekaran** [cerner/carbon-graphs#316]
**What part of Carbon does this bug correlate to?**
General graph logic:

* [x]  Graph attributes (axes, legends, regions...etc.)
* [ ]  Other (please specify): 

Specific graph logic:

* [ ]  Bar graph
* [ ]  Bubble graph
* [ ]  Gantt chart
* [x]  Line graph
* [x]  Paired result
* [ ]  Pie chart
* [ ]  Scatter plot
* [ ]  Timeline

**Describe the bug**
There is a console error in our advanced-graphing application when we use resizeHandler().

**Expected behavior**
Graphs should be resized without any console error in the browser.

**Screenshots**
![image](https://user-images.githubusercontent.com/60426528/98098449-5beae780-1eb4-11eb-9b6b-ad4bbee9ca72.png)

**Comment by JananiGunasekaran** [comment]
@mraplumb @Veera81 I will be looking into this issue. Can you please assign this issue to my name?

**Comment by JananiGunasekaran** [comment]
@mraplumb @Veera81 I have raised PR to fix this issue. Please review. https://github.com/cerner/carbon-graphs/pull/317
